README.md

```diff
@@ -1,13 +1,17 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+Firts node.js project assigment from Technigo 
+Create my own API
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Its a lot of new things to learn and I find it difficult. 
+Got some help from former students and re watching the lectures.
+
+Got stuck trying to make a query for the "title", cant make it work. I commented out the code but will return to have an other look when I have more time.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://dashboard.heroku.com/apps/week17firstnodejs
```

Comment:
You gave the link to your dashboard, not to your app 😢  `https://week17firstnodejs.herokuapp.com/` is your app address